CHANGELOG.md

```diff
@@ -15,6 +15,9 @@ version directory, and then changes are introduced.
 - Switched from cloudinit to ignition.
 - Enable admission plugins: DefaultTolerationSeconds, MutatingAdmissionWebhook, ValidatingAdmissionWebhook.
 
+## [v3.6.1]
```

Comment:
one thing please add hyperlink to the bottom

Reply:
I had too fast finger. I'll add it in the PR with actual change.